src/NewButton/styles.ts

```diff
@@ -217,6 +217,7 @@ export const getButtonStyles = (theme?: Theme) => {
     ...getBaseStyles(theme),
     display: 'grid',
     gridTemplateAreas: '"leadingIcon text trailingIcon"',
+    gridTemplateColumns: 'min-content 1fr min-content',
```

Comment:
If we decide to go with the center-aligned approach I recommended, we can remove this style and just add `justify-content: center;`